custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -29,8 +30,19 @@ class PredictMixin:
 
     """
 
-    def do_predict(self, logger=None):
+    def _predict_or_transform(self, logger=None, predict=True):
         response_status = HTTP_200_OK
+        if predict is False and self._target_type != TargetType.TRANSFORM.value:
+            wrong_target_type_error_message = (
```

Comment:
As I understand, this error message is in case wrong endpoint is used.
Please change the message to contain "endpoint" terminology, e.t. "Your project is transform, use /transform/ endpoint"
"Your project is binary, use /predict/ endpoint"
just handle the case if target_type is unstructured - use 'predictUnstructured' endpoint